packages/jest-changed-files/src/git.js

```diff
@@ -40,9 +40,11 @@ function findChangedFiles(
         if (stdout === '') {
           resolve([]);
         } else {
-          resolve(stdout.split('\n').map(
-            changedPath => path.resolve(cwd, changedPath),
-          ));
+          resolve(
+            stdout
+              .split('\n')
+              .map(changedPath => path.resolve(cwd, changedPath)),
```

Comment:
I'd still rather like this:
```js
stdout.split('\n') // this, not stdout, returns an array so we can .map()
  .map(changedPath => path.resolve(cwd, changedPath)),
```
but I understand it may be unnecessarily complicated to analyse such case. 
As a good practice here we could also assign `stdout.split('\n')` to a variable.